tests/unit/test_utils.py

```diff
@@ -74,3 +78,53 @@ def test_pretty_bytes_nonfixed_units(size, metric, expected_output):
     # test pretty bytes when NOT specifying a fixed unit of measurement
     output = pretty_bytes(size, metric=metric, precision=1)
     assert output == expected_output
+
+
+def test_normalize_path():
+    # Test that path is normalized for macOS, but does nothing for other platforms.
```

Comment:
How about using normal docstrings to document the function (not lines within a function). That's what's used for other newer Vorta code. E.g.

```
def blah_func():
  """
  Lorem ipsum...
  """
```

Reply:
Done 👍